### PR TITLE
Tex: add modern textbf/textit/texttt commands

### PR DESCRIPTION
**Pull request by sanjayankur31**

### CODE REVIEW

**Comment by sanjayankur31** [comment]
Question: is there value in making sure the ultisnip and snippets commands are the same. Or, on the contrary, is there a reason to make sure they're not the same?

**Comment by lpil** [comment]
The ultisnips ones should be removed- they should only exist for specific snippets that cannot be implemented using the snipmate format.

**Comment by lpil** [comment]
It is already documented, but improvements to the README are welcome.

**Comment by sanjayankur31** [comment]
Updated to remove UltiSnip snippets that are already defined in the snipmate format.

I couldn't find where in the docs it says to prefer snipmat since UltiSnips can also read them, so I've added a line in the Readme.

File: UltiSnips/tex.snippets [hunk 1]
```diff
@@ -467,112 +387,4 @@ snippet ac "Acroynm normal" bi
...
-\subparagraph{$1${VISUAL}}
-endsnippet
-
-snippet subpar* "Sub Paragraph*" wi
-\subparagraph*{$1${VISUAL}}
-endsnippet
-
-snippet chapl "Chapter with label" wi
-\chapter{$1${VISUAL}}
-\label{cha:${2:${1/\W+/-/g}}}
```
Comment (sanjayankur31) [inline UltiSnips/tex.snippets:525]: these are already defined in snipmate, so I removed them, but the labels here are more advanced, so we could leave these in as UltiSnips specific overrides.

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**